hack/generate-manifest.sh

```diff
@@ -157,6 +159,11 @@ case $key in
     CUSTOM_ADM_CONTROLLER=true
     shift
     ;;
+    --dev-image)
```

Comment:
I would recommend not adding this new parameter since we already use the `IMG_NAME` env variable. Maybe you can use that one and check if it is set in the code below (instead of `DEV_IMAGE`)?

Reply:
Done, thanks for the pointer.